.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description of change
+
+_Please provide a description of change here._
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items:_
+
+- [ ] Does `make -j8 test` (UNIX) or `vcbuild test nosign` (Windows) pass with
+  this change (including linting)?
+- [ ] Is the commit message formatted according to [CONTRIBUTING.md][0]
+- [ ] Is a regression test (or benchmark if this is a performance-related
```

Comment:
Perhaps, something like this?

```
If this PR fixes a bug (or performance problem), is regression test include (or a benchmark)?
```

I'll update wording just in case if it sounds better.